### DISCUSSION TITLE
Falsche Zählerwerte

### DISCUSSION THREAD

**Discussion by Duke194** [jomjol/AI-on-the-edge-device#3134] [General]
Hallo Ich habe ein ähnliches Problem und hab mir ein paar Gedanken dazu gemacht. Ihr dürft mich gerne korrigieren wenn ich hier einen Gedankenfehler gemacht habe, aber müsste man nicht die Digitalisierten Werte immer mit floor abrunden? 

Ich bin der Meinung, dass die Erkennung im Beispiel alles richtig erkennt, die Zahl jedoch sollte meiner Meinung nach 713.4284
sein, was sie auch wäre, wenn die Zahlen abgerundet werden würden. Das macht denke ich insofern Sinn, dass ja die Zahl dann "abgeschlossen" ist wenn sie mittig in die ROI gedreht ist. Teil mal gerne eure Gedanken dazu. Danke im voraus :)

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/16086934/7d261ee4-d123-4af3-9743-30db928a38bc)

_Originally posted by @Duke194 in https://github.com/jomjol/AI-on-the-edge-device/issues/3028#issuecomment-2207444748_

**Comment by SybexX** [comment]
Es gibt zig verschiedene Wasserzähler mit unterschiedlichem Verhalten (früher, normaler und später Übergang/Drehung der Zahlen) , dass man es nicht so pauschal sagen kann. In deinem Fall mag es zutreffen, jedoch wird es wahrscheinlich bei vielen anderen zu Problemen führen.
Die Erkennung der Zahlen ist nicht immer zu 100% richtig und es wird immer Abweichungen von dem Tatsächlichen Wert geben.
Um diese Ungenauigkeiten auszugleichen muß man halt die Berechnung immer wieder verbessern, bis man das Optimum erreicht hat.

**Reply by Duke194** [reply]
Okey, verstehe. Ich hab im Nachgang auch nochmal etwas akribischer die Digit ROIs gesetzt. Für den Moment scheint das den Großteil der Fehler abzufangen.
